TypeCobol/Compiler/CupParser/NodeBuilder/ProgramClassBuilder.cs

```diff
@@ -523,11 +512,28 @@ public virtual void StartDataRedefinesEntry(DataRedefinesEntry entry)
             Enter(node, null, symbolTable);
             node.SymbolTable.AddVariable(node);
 
+            //Add all index to symbol table
+            AddIndexToSymbolTableAll(entry, symbolTable);
+            
             Dispatcher.StartDataRedefinesEntry(entry);
 
             CheckIfItsTyped(node, node.CodeElement);
         }
 
+        public virtual void AddIndexToSymbolTableAll(CommonDataDescriptionAndDataRedefines entry, SymbolTable symbolTable)
```

Comment:
As this method add Index to SymbolTable but also to the AST, I'd rather rename it CreateIndexAndAddToSymbolTable

Reply:
Done.